x-pack/plugins/apm/public/components/shared/EmptyMessage.tsx

```diff
@@ -4,22 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiEmptyPrompt } from '@elastic/eui';
+import { EuiEmptyPrompt, EuiEmptyPromptProps } from '@elastic/eui';
 import React from 'react';
 
-function EmptyMessage({
+interface Props {
+  heading?: string;
+  subheading?: EuiEmptyPromptProps['body'];
+  hideSubheading?: boolean;
+}
+
+const EmptyMessage: React.SFC<Props> = ({
   heading = 'No data found.',
   subheading = 'Try another time range or reset the search filter.',
   hideSubheading = false
-}) {
+}) => {
   return (
     <EuiEmptyPrompt
       titleSize="s"
       title={<div>{heading || 'No data found.'}</div>}
       body={!hideSubheading && subheading}
     />
   );
-}
+};
 
-// tslint:disable-next-line:no-default-export
-export default EmptyMessage;
+export { EmptyMessage };
```

Comment:
Does EmptyMessage still make sense to have, or should we just use `EuiEmptyPrompt` directly?

Reply:
This probably makes sense, punting on it for now though since I've already gone way overboard overloading this bugfix ticket lol